lib/MTT/Test/Build/Trivial.pm

```diff
@@ -56,6 +58,8 @@ sub Build {
                        "trivial_tests_cflags");
     my $fflags = Value($ini, $config->{full_section_name}, 
                        "trivial_tests_fflags");
+    my $f08flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f08flags");
```

Comment:
so what is the spec of usempif08_flags ?
does it override fflags ? or should it be appended to fflags for f08 programs only ?